### PR TITLE
Backend/posts

### PR DESCRIPTION
**Pull request by himokkk**

### CODE REVIEW

File: Social_Networking_Platform/settings.py [hunk 1]
```diff
@@ -22,13 +22,15 @@
...
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-0d^xt!0ryogr0)*^+37(sl2egph&$drxs+e!6xz!b2-tt)u4l#'
+SECRET_KEY = "django-insecure-0d^xt!0ryogr0)*^+37(sl2egph&$drxs+e!6xz!b2-tt)u4l#"
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
 ALLOWED_HOSTS = []
 
+POST_LIKE_INTERACTIONS = ["like", "unlike"]
```
Comment (himokkk) [inline Social_Networking_Platform/settings.py:32]: ???

File: Social_Networking_Platform/settings.py [hunk 2]
```diff
@@ -99,6 +109,7 @@
 AUTH_PASSWORD_VALIDATORS = [
     {
         "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
+        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
```
Comment (himokkk) [inline Social_Networking_Platform/settings.py:112]: useful

File: posts/serializers.py [hunk 3]
```diff
@@ -14,8 +25,34 @@ def get_author_username(self, obj) -> str | None:
...
+        if obj.comments:
+            return obj.comments.count()
 
-class CreatePostSerializer(serializers.ModelSerializer):
+
+class PostLikeSerializer(serializers.ModelSerializer):
     class Meta:
         model = Post
-        fields = ("content", "media")
+        fields = ()
```
Comment (himokkk) [inline posts/serializers.py:40]: what doing?

**Comment by mldxo** [comment]
not too many changes to be honest

**Pull request merged**